### PR TITLE
Attempt to fix td.

### PR DESCRIPTION
**Pull request by p0psicles**
But the html is not parsing.

- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by p0psicles** [comment]
Should we move it to html folder now?

File: medusa/providers/torrent/json/torrentday.py [hunk 1]
```diff
@@ -147,21 +145,23 @@ def process_column_header(td):
...
             # Skip column headers
             for row in torrent_rows[1:]:
                 try:
-                    name = row.find('td')[labels.index('Name')]
+                    name = row('td')[labels.index('name')]
                     title = name.find('a').get_text(strip=True)
                     # details = name.find('a')['href']
-                    download_url = row.find('td')[labels.index('Download')]
+                    download_url_raw = row('td')[labels.index('download')].find('a')['href']
+                    download_url = urljoin(self.url, download_url_raw)
```
Comment (medariox) [inline medusa/providers/torrent/json/torrentday.py]: Please join after the if check.

**Comment by medariox** [comment]
Yes, I think we should.

**Comment by p0psicles** [comment]
Getting error that downloaded torrent is not valid.

**Pull request merged**